hw1/task1/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Task1
+{
+    /// <summary>Internal class Program.</summary>
+    internal class Program
+    {
+        /// <summary>Function that called when running application.</summary>
+        /// <param name="args">Command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            if (args.Length == 0 || args[0] == "help")
+            {
+                Console.WriteLine("Pass array of integers as command arguments (one number per argument) to sort it");
+                Console.WriteLine("For example: task1 3 4 1 5 6");
+                Console.WriteLine("Or: dotnet run 10 9 8 7 6 ");
+                return;
+            }
+
+            int[] arr = new int[args.Length];
```

Comment:
```suggestion
            var arr = new int[args.Length];
```
Тип очевиден из правой части